Lib/pdb.py

```diff
@@ -162,18 +162,18 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, skip=None,
         # Read $HOME/.pdbrc and ./.pdbrc
         self.rcLines = []
         if readrc:
+            readrcFiles = []
```

Comment:
Small neat: please use `name_with_underscores` instead of `camelCase`

Reply:
@asvetlov 
I've tried to keep the style of the other vars there.
Can change it, but want to make sure it is really wanted.

Reply:
`self.rcLines` exists starting from 2001.
Too late to change the name, it is a part of public API for good or for bad.
But for new variables better to follow PEP 8 style.